src/WebJobs.Script.WebHost/web.InProcess.xdt

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
+  <system.webServer>
+    <handlers>
+      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModuleV2" resourceType="Unspecified" xdt:Transform="Replace" xdt:Locator="Match(name)" />
+    </handlers>
+    <aspNetCore processPath=".\Microsoft.Azure.WebJobs.Script.WebHost.exe" arguments="" stdoutLogEnabled="true" stdoutLogFile="d:\home\logfiles\stdout" hostingModel="inprocess" xdt:Transform="Replace" xdt:Locator="Match(processPath)" />
```

Comment:
We had explicitly disabled stdout logging in the past, do we need to have it enabled? This will cause files to be written to the slow file system.